sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Add-student-id-sfas-individuals.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+    sims.sfas_individuals
+ADD
+    COLUMN IF NOT EXISTS student_id INTEGER;
+
+COMMENT ON COLUMN sims.sfas_individuals.student_id IS 'Student id used to import data to sims.disbursement_overawards table.';
+
+ALTER TABLE
```

Comment:
Please take a look at existing relationships, usually, we just add them alongside the column declaration. This will also enforce the proper naming conventions. The `fk_student_id_id` is not aligned with the existing ones.
```sql
ALTER TABLE
    sims.sfas_individuals
ADD
    COLUMN IF NOT EXISTS student_id INTEGER REFERENCES sims.students(id);
```